docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +79,19 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+
+.. _intro:troubleshooting:installation:postgresql-autodetect-issues:
+
+Autodetection of the PostgreSQL setup
+.....................................
+
+Sometimes AiiDA fails to autodetect the local configuration of PostgreSQL when running ``verdi quicksetup``.
+In that case try to:
+
+    1. Create the database manually in PostgreSQL (see :ref:`here<intro:install:database>`)
+    2. Then run the full ``verdi setup`` command (see :ref:`here<intro:install:verdi_setup>`)
```

Comment:
Just adding periods since those are sentences.
```suggestion
    1. Create the database manually in PostgreSQL (see :ref:`here<intro:install:database>`).
    2. Then run the full ``verdi setup`` command (see :ref:`here<intro:install:verdi_setup>`).
```

Reply:
Yeah I never know when to punctuate list elements or not =S